bot/exts/christmas/hanukkah_embed.py

```diff
@@ -55,48 +58,57 @@ async def hanukkah_festival(self, ctx: commands.Context) -> None:
         month = str(today.month)
         year = str(today.year)
         embed = Embed()
-        embed.title = 'Hanukkah'
+        embed.title = "Hanukkah"
         embed.colour = Colours.blue
         if day in self.hanukkah_days and month in self.hanukkah_months and year in self.hanukkah_years:
             if int(day) == hanukkah_start_day:
                 now = datetime.datetime.utcnow()
-                now = str(now)
-                hours = int(now[11:13]) + 4  # using only hours
+                hours = now.hour + 4  # using only hours
                 hanukkah_start_hour = 18
                 if hours < hanukkah_start_hour:
-                    embed.description = (f"Hanukkah hasnt started yet, "
-                                         f"it will start in about {hanukkah_start_hour-hours} hour/s.")
-                    return await ctx.send(embed=embed)
+                    embed.description = (
+                        "Hanukkah hasnt started yet, "
+                        f"it will start in about {hanukkah_start_hour - hours} hour/s."
+                    )
+                    await ctx.send(embed=embed)
+                    return
                 elif hours > hanukkah_start_hour:
-                    embed.description = (f'It is the starting day of Hanukkah ! '
-                                         f'Its been {hours-hanukkah_start_hour} hours hanukkah started !')
-                    return await ctx.send(embed=embed)
+                    embed.description = (
+                        "It is the starting day of Hanukkah ! "
+                        f"Its been {hours - hanukkah_start_hour} hours hanukkah started !"
+                    )
+                    await ctx.send(embed=embed)
+                    return
             festival_day = self.hanukkah_days.index(day)
-            number_suffixes = ['st', 'nd', 'rd', 'th']
-            suffix = ''
-            if int(festival_day) == 1:
+            number_suffixes = ["st", "nd", "rd", "th"]
+            suffix = ""
+            if festival_day == 1:
                 suffix = number_suffixes[0]
-            if int(festival_day) == 2:
+            if festival_day == 2:
                 suffix = number_suffixes[1]
-            if int(festival_day) == 3:
+            if festival_day == 3:
                 suffix = number_suffixes[2]
-            if int(festival_day) > 3:
+            if festival_day > 3:
                 suffix = number_suffixes[3]
```

Comment:
Can't this just be
```py
suffix = number_suffixes[festival_day - 1 if festival_day <= 3 else 3]
``` 
(or something more readable than what I just wrote)
(I also don't see why `suffix` is being defined as an empty string before this either)